src/mudlet.cpp

```diff
@@ -407,8 +411,11 @@ mudlet::mudlet()
     mpMainToolBar->widgetForAction(mpActionReconnect)->setObjectName(mpActionReconnect->objectName());
 
     mpActionMultiView = new QAction(QIcon(QStringLiteral(":/icons/view-split-left-right.png")), tr("MultiView"), this);
-    mpActionMultiView->setToolTip(tr("If you've got multiple profiles open, splits Mudlet screen to show them all at once"));
+    mpActionMultiView->setToolTip(tr("<p>Enabled when multiple profiles are open, this splits the Mudlet screen to show them all at once.</p>"));
```

Comment:
The text seems a bit convoluted now. Split into two sentences.
Do not give HTML tags to translators unless utterly necessary.

```suggestion
    mpActionMultiView->setToolTip(QString("<p>%1</p>").arg(tr("Splits the Mudlet screen to show multiple profiles at once. Deactivated while you open less.")));
```
Code has not been tested, just the general idea here.

Reply:
You didn't use `QStringLiteral(`...`)` so that will still be marked as an ambiguous `QString` - there is a `(static QString) mudlet::htmlWrapper` defined currently as `QStringLiteral("<html><head /><body>%1</body></html>")` which I put in a while back to cover this exact situation. However, since then I found it should be trimmed to be `QStringLiteral("<p>%1</p>")` - but then it would mean some other unrelated tooltips would have to have existing `<p>`...`</p>` taken out from their Engineering English text - thus making this PR even larger...

I was also in two minds as whether it is cleaner to leave the bracketing `<p>`...`</p>` in place so the in the odd occasion where more than one paragraph is wanted - to separate, perhaps a significant detail - the translator is not presented with an odd looking `</p><p>` in the **middle** of a text *without* the matching tags at either end - which have been concealed from them - even if it doesn't confuse them I am not sure that the CrowdIn auto-hiding of tags is going to make sense of those...

Reply:
`"Splits the Mudlet screen to show multiple profiles at once; disabled when less than two are loaded."`?

"Deactivated while you open less." is not grammatically correct IMHO - though I do not know if I can explain why...

Also I am working on another PR that could get the user back to the state as if they have cancelled the initial "Open a profile" dialogue so increasing the possibility of not having ANY profiles loaded/connected...

Reply:
Fair enough. Not sure why you prefer semicolon to seperate sentences but meh.

Reply:
Dammit - I've been rumbled; semicolons enable me to keep going for longer without having to pause or otherwise interrupt my narrative by coming to a halt, period (or, if you like) a full stop **.**